packages/react/src/button/Button.tsx

```diff
@@ -15,38 +15,70 @@ import { Stack } from "../stack";
 import { Text } from "../text";
 import { type Recipe, recipe } from "./Button.recipe";
 
+const presets = {
+  basic: { colorScheme: "secondary", variant: "solid" },
+  basicLink: { colorScheme: "secondary", variant: "link" },
```

Comment:
let's use dash case for prop values
```suggestion
  "basic-link": { colorScheme: "secondary", variant: "link" },
```